MMM-cryptocurrency.js

```diff
@@ -509,5 +404,1010 @@ Module.register('MMM-cryptocurrency', {
             sv: 'translations/sv.json'
         }
     },
+    
+    setSparklineIds: function(){
+        this.sparklineIds = {
```

Comment:
The best possible would be to have these in a separate file.. But I can do later.
Great job!